Sources/SwerverCore/Router/Router.swift

```diff
@@ -19,26 +17,21 @@ public class Router {
 
         if routes.routeExist(url: targetURL) {
 
-            let allowedActions = routes.fetchAllActions(url: targetURL)
-
-            if isOptionsRequest(request: request) {
-                return responseHeaderFormatter.format(httpResponse: routes.optionsAction(url: targetURL).execute())
-            }
+            let targetRoute = routes.fetchRoute(url: targetURL)
 
-            guard let action = allowedActions[targetMethod] else {
+            guard let action = targetRoute[targetMethod] else {
                 return "HTTP/1.1"
             }
             return responseHeaderFormatter.format(httpResponse: action.execute())
         }
         return responseHeaderFormatter.format(httpResponse: notFoundAction.execute())
     }
 
-    private func isOptionsRequest(request: HttpRequest) -> Bool {
-        return request.getMethod() == RequestMethod.options
-    }
-
     private func populateRoutes() {
```

Comment:
We should discuss how you are doing routing.

Reply:
ok

Reply:
noted in trello card